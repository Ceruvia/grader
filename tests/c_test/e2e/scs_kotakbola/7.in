679 710 674
M 32
M 41
H 6
B 68
B 76
H 39
M 15
B 28
M 29
H 78
M 75
M 81
M 39
H 79
B 86
H 44
M 83
B 15
B 51
B 30
M 80
M 45
M 1
B 90
M 73
B 36
M 44
B 80
E
