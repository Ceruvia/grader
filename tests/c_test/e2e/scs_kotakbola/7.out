246 638 560
Ya
